www/StreamingMedia.js

```diff
@@ -1,39 +1,39 @@
-"use strict";

```

Comment:
Please fix this file so it isn't completely recommitted. I can't see a diff here